plumpy/processes.py

```diff
@@ -1318,6 +1318,5 @@ def get_status_info(self, out_status_info: dict) -> None:
             'ctime': self.creation_time,
             'paused': self.paused,
             'process_string': str(self),
-            'state': self.state,
-            'state_info': str(self._state)
```

Comment:
not sure whether this is considered part of the API; you'll know best which version to bump

Reply:
Technically it is part of the API but I doubt that anyone is using it. We are not even using it in `aiida-core`. Anyway, I think these changes require a new minor version and since we are pre-v1.0 minor versions can be expected to introduce changes in API, so I think this is fine to do.